### DISCUSSION TITLE
no bootable device

### DISCUSSION THREAD

**Discussion by wuge0** [utmapp/UTM#2415] [UTM Help (iOS)]
-[CSConnection init]:273
2021-03-26 10:48:24.525 UTM[7471:120389] -[CSConnection init]:273
Failed to access security scoped resource for: file:///private/var/containers/Bundle/Application/716E6CC5-4747-4FD4-9BFA-D18B77579B98/UTM.app/qemu/
2021-03-26 10:48:24.547 UTM[7471:120389] Failed to access security scoped resource for: file:///private/var/containers/Bundle/Application/716E6CC5-4747-4FD4-9BFA-D18B77579B98/UTM.app/qemu/
Running:  -L /var/containers/Bundle/Application/716E6CC5-4747-4FD4-9BFA-D18B77579B98/UTM.app/qemu -S -qmp tcp:127.0.0.1:4000,server,nowait -vga none -spice port=4001,addr=127.0.0.1,disable-ticketing,image-compression=off,playback-compression=off,streaming-video=off -device qxl-vga -smp cpus=1,sockets=1,cores=1,threads=1 -machine pc, -accel tcg,tb-size=256,split-wx=on -global PIIX4_PM.disable_s3=1 -global ICH9-LPC.disable_s3=1 -boot order=d -m 1024 -device AC97 -name "Virtual Machine 1" -usb -device usb-tablet -device usb-mouse -device usb-kbd -device ide-hd,bus=ide.0,drive=drive0,bootindex=0 -drive "if=none,media=disk,id=drive0,file=/private/var/mobile/Containers/Data/Application/0190FAA4-BB6B-43E7-83B3-7486B61F6486/Documents/Virtual Machine 1.utm/Images/zh-hans_windows_xp_professional_with_service_pack_3_x86_cd_x14-80404.iso,cache=writethrough" -device ide-hd,bus=ide.1,drive=drive1,bootindex=1 -drive "if=none,media=disk,id=drive1,file=/private/var/mobile/Containers/Data/Application/0190FAA4-BB6B-43E7-83B3-7486B61F6486/Documents/Virtual Machine 1.utm/Images/had.img,cache=writethrough" -device rtl8139,mac=B6:5B:C2:33:D8:FA,netdev=net0 -netdev user,id=net0 -device virtio-serial -device virtserialport,chardev=vdagent,name=com.redhat.spice.0 -chardev spicevmc,id=vdagent,debug=0,name=vdagent -uuid 2EED0E97-C248-4A53-9D3D-52C2C55A9765 -rtc base=localtime
Loading libqemu-i386-softmmu.utm.dylib
2021-03-26 10:48:24.592 UTM[7471:120389] Running:  -L /var/containers/Bundle/Application/716E6CC5-4747-4FD4-9BFA-D18B77579B98/UTM.app/qemu -S -qmp tcp:127.0.0.1:4000,server,nowait -vga none -spice port=4001,addr=127.0.0.1,disable-ticketing,image-compression=off,playback-compression=off,streaming-video=off -device qxl-vga -smp cpus=1,sockets=1,cores=1,threads=1 -machine pc, -accel tcg,tb-size=256,split-wx=on -global PIIX4_PM.disable_s3=1 -global ICH9-LPC.disable_s3=1 -boot order=d -m 1024 -device AC97 -name "Virtual Machine 1" -usb -device usb-tablet -device usb-mouse -device usb-kbd -device ide-hd,bus=ide.0,drive=drive0,bootindex=0 -drive "if=none,media=disk,id=drive0,file=/private/var/mobile/Containers/Data/Application/0190FAA4-BB6B-43E7-83B3-7486B61F6486/Documents/Virtual Machine 1.utm/Images/zh-hans_windows_xp_professional_with_service_pack_3_x86_cd_x14-80404.iso,cache=writethrough" -device ide-hd,bus=ide.1,drive=drive1,bootindex=1 -drive "if=none,media=disk,id=drive1,file=/private/var/mobile/Containers/Data/Application/0190FAA4-BB6B-43E7-83B3-7486B61F6486/Documents/Virtual Machine 1.utm/Images/had.img,cache=writethrough" -device rtl8139,mac=B6:5B:C2:33:D8:FA,netdev=net0 -netdev user,id=net0 -device virtio-serial -device virtserialport,chardev=vdagent,name=com.redhat.spice.0 -chardev spicevmc,id=vdagent,debug=0,name=vdagent -uuid 2EED0E97-C248-4A53-9D3D-52C2C55A9765 -rtc base=localtime
2021-03-26 10:48:24.592 UTM[7471:120389] Loading libqemu-i386-softmmu.utm.dylib
SPICE port not in use yet, retries left: 29
2021-03-26 10:48:24.623 UTM[7471:120394] SPICE port not in use yet, retries left: 29
Stream error Error Domain=NSPOSIXErrorDomain Code=61 "Connection refused" UserInfo={_kCFStreamErrorCodeKey=61, _kCFStreamErrorDomainKey=1}
QMP stream error seen: Error Domain=NSPOSIXErrorDomain Code=61 "Connection refused" UserInfo={_kCFStreamErrorCodeKey=61, _kCFStreamErrorDomainKey=1}
QMP connection failed, retries left: 29
2021-03-26 10:48:24.627 UTM[7471:120398] Stream error Error Domain=NSPOSIXErrorDomain Code=61 "Connection refused" UserInfo={_kCFStreamErrorCodeKey=61, _kCFStreamErrorDomainKey=1}
2021-03-26 10:48:24.627 UTM[7471:120398] QMP stream error seen: Error Domain=NSPOSIXErrorDomain Code=61 "Connection refused" UserInfo={_kCFStreamErrorCodeKey=61, _kCFStreamErrorDomainKey=1}
2021-03-26 10:48:24.627 UTM[7471:120398] QMP connection failed, retries left: 29
WARNING: Image format was not specified for '/private/var/mobile/Containers/Data/Application/0190FAA4-BB6B-43E7-83B3-7486B61F6486/Documents/Virtual Machine 1.utm/Images/zh-hans_windows_xp_professional_with_service_pack_3_x86_cd_x14-80404.iso' and probing guessed raw.
         Automatically detecting the format is dangerous for raw images, write operations on block 0 will be restricted.
         Specify the 'raw' format explicitly to remove the restrictions.
Connected to stream
QMP connection successful! (readStream:1)
2021-03-26 10:48:25.639 UTM[7471:120394] Connected to stream
2021-03-26 10:48:25.639 UTM[7471:120394] QMP connection successful! (readStream:1)
Debug JSON recieved <- {
    QMP =     {
        capabilities =         (
            oob
        );
        version =         {
            package = "";
            qemu =             {
                major = 5;
                micro = 0;
                minor = 2;
            };
        };
    };
}
Connected to stream
2021-03-26 10:48:25.641 UTM[7471:120394] Debug JSON recieved <- {
    QMP =     {
        capabilities =         (
            oob
        );
        version =         {
            package = "";
            qemu =             {
                major = 5;
                micro = 0;
                minor = 2;
            };
        };
    };
}
QMP connection successful! (readStream:0)
2021-03-26 10:48:25.641 UTM[7471:120394] Connected to stream
2021-03-26 10:48:25.641 UTM[7471:120394] QMP connection successful! (readStream:0)
Got QMP handshake: {
    QMP =     {
        capabilities =         (
            oob
        );
        version =         {
            package = "";
            qemu =             {
                major = 5;
                micro = 0;
                minor = 2;
            };
        };
    };
}
Debug JSON send -> {
    execute = "qmp_capabilities";
}
2021-03-26 10:48:25.642 UTM[7471:120454] Got QMP handshake: {
    QMP =     {
        capabilities =         (
            oob
        );
        version =         {
            package = "";
            qemu =             {
                major = 5;
                micro = 0;
                minor = 2;
            };
        };
    };
}
2021-03-26 10:48:25.642 UTM[7471:120454] Debug JSON send -> {
    execute = "qmp_capabilities";
}
-[CSSession initWithSession:]:294
2021-03-26 10:48:25.644 UTM[7471:120456] -[CSSession initWithSession:]:294
shared directory disabled
2021-03-26 10:48:25.645 UTM[7471:120456] shared directory disabled
Debug JSON recieved <- {
    return =     {
    };
}
2021-03-26 10:48:25.646 UTM[7471:120394] Debug JSON recieved <- {
    return =     {
    };
}
qemuQmpDidConnect
2021-03-26 10:48:25.648 UTM[7471:120454] qemuQmpDidConnect
Debug JSON send -> {
    execute = cont;
}
2021-03-26 10:48:25.648 UTM[7471:120454] Debug JSON send -> {
    execute = cont;
}
Debug JSON recieved <- {
    event = RESUME;
    timestamp =     {
        microseconds = 652294;
        seconds = 1616726905;
    };
}
2021-03-26 10:48:25.653 UTM[7471:120398] Debug JSON recieved <- {
    event = RESUME;
    timestamp =     {
        microseconds = 652294;
        seconds = 1616726905;
    };
}
Debug JSON recieved <- {
    return =     {
    };
}
2021-03-26 10:48:25.653 UTM[7471:120398] Debug JSON recieved <- {
    return =     {
    };
}
qemuHasResumed
2021-03-26 10:48:25.655 UTM[7471:120394] qemuHasResumed
cs_channel_new:159
2021-03-26 10:48:25.663 UTM[7471:120456] cs_channel_new:159
Debug JSON recieved <- {
    data =     {
        client =         {
            family = ipv4;
            host = "127.0.0.1";
            port = 55178;
        };
        server =         {
            family = ipv4;
            host = "127.0.0.1";
            port = 4001;
        };
    };
    event = "SPICE_CONNECTED";
    timestamp =     {
        microseconds = 668667;
        seconds = 1616726905;
    };
}
2021-03-26 10:48:25.669 UTM[7471:120454] Debug JSON recieved <- {
    data =     {
        client =         {
            family = ipv4;
            host = "127.0.0.1";
            port = 55178;
        };
        server =         {
            family = ipv4;
            host = "127.0.0.1";
            port = 4001;
        };
    };
    event = "SPICE_CONNECTED";
    timestamp =     {
        microseconds = 668667;
        seconds = 1616726905;
    };
}
Debug JSON recieved <- {
    data =     {
        client =         {
            "channel-id" = 0;
            "channel-type" = 1;
            "connection-id" = 16807;
            family = ipv4;
            host = "127.0.0.1";
            port = 55178;
            tls = 0;
        };
        server =         {
            auth = none;
            family = ipv4;
            host = "127.0.0.1";
            port = 4001;
        };
    };
    event = "SPICE_INITIALIZED";
    timestamp =     {
        microseconds = 713653;
        seconds = 1616726905;
    };
}
qemu-system: info: main channel: opened
2021-03-26 10:48:25.714 UTM[7471:120454] Debug JSON recieved <- {
    data =     {
        client =         {
            "channel-id" = 0;
            "channel-type" = 1;
            "connection-id" = 16807;
            family = ipv4;
            host = "127.0.0.1";
            port = 55178;
            tls = 0;
        };
        server =         {
            auth = none;
            family = ipv4;
            host = "127.0.0.1";
            port = 4001;
        };
    };
    event = "SPICE_INITIALIZED";
    timestamp =     {
        microseconds = 713653;
        seconds = 1616726905;
    };
}
cs_channel_new:168
Debug JSON recieved <- {
    data =     {
        client =         {
            family = ipv4;
            host = "127.0.0.1";
            port = 55179;
        };
        server =         {
            family = ipv4;
            host = "127.0.0.1";
            port = 4001;
        };
    };
    event = "SPICE_CONNECTED";
    timestamp =     {
        microseconds = 717618;
        seconds = 1616726905;
    };
}
2021-03-26 10:48:25.716 UTM[7471:120436] cs_channel_new:168
2021-03-26 10:48:25.718 UTM[7471:120456] Debug JSON recieved <- {
    data =     {
        client =         {
            family = ipv4;
            host = "127.0.0.1";
            port = 55179;
        };
        server =         {
            family = ipv4;
            host = "127.0.0.1";
            port = 4001;
        };
    };
    event = "SPICE_CONNECTED";
    timestamp =     {
        microseconds = 717618;
        seconds = 1616726905;
    };
}
Debug JSON recieved <- {
    data =     {
        client =         {
            family = ipv4;
            host = "127.0.0.1";
            port = 55180;
        };
        server =         {
            family = ipv4;
            host = "127.0.0.1";
            port = 4001;
        };
    };
    event = "SPICE_CONNECTED";
    timestamp =     {
        microseconds = 718485;
        seconds = 1616726905;
    };
}
2021-03-26 10:48:25.718 UTM[7471:120394] Debug JSON recieved <- {
    data =     {
        client =         {
            family = ipv4;
            host = "127.0.0.1";
            port = 55180;
        };
        server =         {
            family = ipv4;
            host = "127.0.0.1";
            port = 4001;
        };
    };
    event = "SPICE_CONNECTED";
    timestamp =     {
        microseconds = 718485;
        seconds = 1616726905;
    };
}
Debug JSON recieved <- {
    data =     {
        client =         {
            family = ipv4;
            host = "127.0.0.1";
            port = 55181;
        };
        server =         {
            family = ipv4;
            host = "127.0.0.1";
            port = 4001;
        };
    };
    event = "SPICE_CONNECTED";
    timestamp =     {
        microseconds = 719104;
        seconds = 1616726905;
    };
}
2021-03-26 10:48:25.719 UTM[7471:120456] Debug JSON recieved <- {
    data =     {
        client =         {
            family = ipv4;
            host = "127.0.0.1";
            port = 55181;
        };
        server =         {
            family = ipv4;
            host = "127.0.0.1";
            port = 4001;
        };
    };
    event = "SPICE_CONNECTED";
    timestamp =     {
        microseconds = 719104;
        seconds = 1616726905;
    };
}
Debug JSON recieved <- {
    data =     {
        client =         {
            "channel-id" = 0;
            "channel-type" = 2;
            "connection-id" = 16807;
            family = ipv4;
            host = "127.0.0.1";
            port = 55181;
            tls = 0;
        };
        server =         {
            auth = none;
            family = ipv4;
            host = "127.0.0.1";
            port = 4001;
        };
    };
    event = "SPICE_INITIALIZED";
    timestamp =     {
        microseconds = 809596;
        seconds = 1616726905;
    };
}
2021-03-26 10:48:25.810 UTM[7471:120394] Debug JSON recieved <- {
    data =     {
        client =         {
            "channel-id" = 0;
            "channel-type" = 2;
            "connection-id" = 16807;
            family = ipv4;
            host = "127.0.0.1";
            port = 55181;
            tls = 0;
        };
        server =         {
            auth = none;
            family = ipv4;
            host = "127.0.0.1";
            port = 4001;
        };
    };
    event = "SPICE_INITIALIZED";
    timestamp =     {
        microseconds = 809596;
        seconds = 1616726905;
    };
}
Debug JSON recieved <- {
    data =     {
        client =         {
            "channel-id" = 0;
            "channel-type" = 5;
            "connection-id" = 16807;
            family = ipv4;
            host = "127.0.0.1";
            port = 55179;
            tls = 0;
        };
        server =         {
            auth = none;
            family = ipv4;
            host = "127.0.0.1";
            port = 4001;
        };
    };
    event = "SPICE_INITIALIZED";
    timestamp =     {
        microseconds = 813238;
        seconds = 1616726905;
    };
}
2021-03-26 10:48:25.813 UTM[7471:120454] Debug JSON recieved <- {
    data =     {
        client =         {
            "channel-id" = 0;
            "channel-type" = 5;
            "connection-id" = 16807;
            family = ipv4;
            host = "127.0.0.1";
            port = 55179;
            tls = 0;
        };
        server =         {
            auth = none;
            family = ipv4;
            host = "127.0.0.1";
            port = 4001;
        };
    };
    event = "SPICE_INITIALIZED";
    timestamp =     {
        microseconds = 813238;
        seconds = 1616726905;
    };
}
-[CSDisplayMetal initWithSession:channelID:monitorID:]:304
cs_channel_new:178
-[CSInput initWithSession:channelID:monitorID:]:435
Debug JSON recieved <- {
    data =     {
        client =         {
            "channel-id" = 0;
            "channel-type" = 6;
            "connection-id" = 16807;
            family = ipv4;
            host = "127.0.0.1";
            port = 55180;
            tls = 0;
        };
        server =         {
            auth = none;
            family = ipv4;
            host = "127.0.0.1";
            port = 4001;
        };
    };
    event = "SPICE_INITIALIZED";
    timestamp =     {
        microseconds = 817187;
        seconds = 1616726905;
    };
}
2021-03-26 10:48:25.816 UTM[7471:120436] -[CSDisplayMetal initWithSession:channelID:monitorID:]:304
2021-03-26 10:48:25.816 UTM[7471:120436] cs_channel_new:178
2021-03-26 10:48:25.816 UTM[7471:120436] -[CSInput initWithSession:channelID:monitorID:]:435
2021-03-26 10:48:25.817 UTM[7471:120456] Debug JSON recieved <- {
    data =     {
        client =         {
            "channel-id" = 0;
            "channel-type" = 6;
            "connection-id" = 16807;
            family = ipv4;
            host = "127.0.0.1";
            port = 55180;
            tls = 0;
        };
        server =         {
            auth = none;
            family = ipv4;
            host = "127.0.0.1";
            port = 4001;
        };
    };
    event = "SPICE_INITIALIZED";
    timestamp =     {
        microseconds = 817187;
        seconds = 1616726905;
    };
}
Debug JSON recieved <- {
    data =     {
        client =         {
            family = ipv4;
            host = "127.0.0.1";
            port = 55182;
        };
        server =         {
            family = ipv4;
            host = "127.0.0.1";
            port = 4001;
        };
    };
    event = "SPICE_CONNECTED";
    timestamp =     {
        microseconds = 818987;
        seconds = 1616726905;
    };
}
2021-03-26 10:48:25.819 UTM[7471:120454] Debug JSON recieved <- {
    data =     {
        client =         {
            family = ipv4;
            host = "127.0.0.1";
            port = 55182;
        };
        server =         {
            family = ipv4;
            host = "127.0.0.1";
            port = 4001;
        };
    };
    event = "SPICE_CONNECTED";
    timestamp =     {
        microseconds = 818987;
        seconds = 1616726905;
    };
}
resizing to (1366.000000, 1024.000000)
2021-03-26 10:48:25.819 UTM[7471:120389] resizing to (1366.000000, 1024.000000)
Debug JSON recieved <- {
    data =     {
        client =         {
            family = ipv4;
            host = "127.0.0.1";
            port = 55183;
        };
        server =         {
            family = ipv4;
            host = "127.0.0.1";
            port = 4001;
        };
    };
    event = "SPICE_CONNECTED";
    timestamp =     {
        microseconds = 820103;
        seconds = 1616726905;
    };
}
qemu-system: GSpice: spice_main_channel_send_monitor_config: assertion 'c->agent_connected' failed
2021-03-26 10:48:25.820 UTM[7471:120456] Debug JSON recieved <- {
    data =     {
        client =         {
            family = ipv4;
            host = "127.0.0.1";
            port = 55183;
        };
        server =         {
            family = ipv4;
            host = "127.0.0.1";
            port = 4001;
        };
    };
    event = "SPICE_CONNECTED";
    timestamp =     {
        microseconds = 820103;
        seconds = 1616726905;
    };
}
Debug JSON recieved <- {
    data =     {
        client =         {
            "channel-id" = 0;
            "channel-type" = 3;
            "connection-id" = 16807;
            family = ipv4;
            host = "127.0.0.1";
            port = 55182;
            tls = 0;
        };
        server =         {
            auth = none;
            family = ipv4;
            host = "127.0.0.1";
            port = 4001;
        };
    };
    event = "SPICE_INITIALIZED";
    timestamp =     {
        microseconds = 885511;
        seconds = 1616726905;
    };
}
2021-03-26 10:48:25.886 UTM[7471:120456] Debug JSON recieved <- {
    data =     {
        client =         {
            "channel-id" = 0;
            "channel-type" = 3;
            "connection-id" = 16807;
            family = ipv4;
            host = "127.0.0.1";
            port = 55182;
            tls = 0;
        };
        server =         {
            auth = none;
            family = ipv4;
            host = "127.0.0.1";
            port = 4001;
        };
    };
    event = "SPICE_INITIALIZED";
    timestamp =     {
        microseconds = 885511;
        seconds = 1616726905;
    };
}
Debug JSON recieved <- {
    data =     {
        client =         {
            "channel-id" = 0;
            "channel-type" = 4;
            "connection-id" = 16807;
            family = ipv4;
            host = "127.0.0.1";
            port = 55183;
            tls = 0;
        };
        server =         {
            auth = none;
            family = ipv4;
            host = "127.0.0.1";
            port = 4001;
        };
    };
    event = "SPICE_INITIALIZED";
    timestamp =     {
        microseconds = 889634;
        seconds = 1616726905;
    };
}
2021-03-26 10:48:25.890 UTM[7471:120394] Debug JSON recieved <- {
    data =     {
        client =         {
            "channel-id" = 0;
            "channel-type" = 4;
            "connection-id" = 16807;
            family = ipv4;
            host = "127.0.0.1";
            port = 55183;
            tls = 0;
        };
        server =         {
            auth = none;
            family = ipv4;
            host = "127.0.0.1";
            port = 4001;
        };
    };
    event = "SPICE_INITIALIZED";
    timestamp =     {
        microseconds = 889634;
        seconds = 1616726905;
    };
}
Debug JSON send -> {
    execute = quit;
}
2021-03-26 10:48:42.899 UTM[7471:120479] Debug JSON send -> {
    execute = quit;
}
Debug JSON recieved <- {
    return =     {
    };
}
Debug JSON recieved <- {
    data =     {
        guest = 0;
        reason = "host-qmp-quit";
    };
    event = SHUTDOWN;
    timestamp =     {
        microseconds = 900459;
        seconds = 1616726922;
    };
}
2021-03-26 10:48:42.900 UTM[7471:120523] Debug JSON recieved <- {
    return =     {
    };
}
2021-03-26 10:48:42.900 UTM[7471:120523] Debug JSON recieved <- {
    data =     {
        guest = 0;
        reason = "host-qmp-quit";
    };
    event = SHUTDOWN;
    timestamp =     {
        microseconds = 900459;
        seconds = 1616726922;
    };
}
qemuWillQuit, reason = host-qmp-quit
2021-03-26 10:48:42.901 UTM[7471:120523] qemuWillQuit, reason = host-qmp-quit
cs_channel_destroy:205
2021-03-26 10:48:42.908 UTM[7471:120436] cs_channel_destroy:205
cs_channel_destroy:222
cs_channel_destroy:197
2021-03-26 10:48:42.908 UTM[7471:120436] cs_channel_destroy:222
2021-03-26 10:48:42.908 UTM[7471:120436] cs_channel_destroy:197
qemu-system: GSpice: Source ID 1 was not found when attempting to remove it

**Comment by StupidRepo** [comment]
### This issue author seems to be O3x777#2770 on the U™ discord server. This is still waiting to be verified.
### Sorry for that annoying and stupid UTM joke, I'm so used to saying U™ now :rofl:

Hello there, please upload your log as a text next time and edit the post to have some description so we don't have to scroll A MILLION MILES DOWN BELOW 😲

In the U™ DC server, you (although its not verified that it's you yet) have said that you're on an `iPhone 11 Pro` and the Installer ISO (or Boot ISO, whatever it is, I don't care.) is an `ARM64` type.

Please go to your VM and press the `EDIT` button (Note: I run U™ on macOS, I don't know how iOS U™ works) and go to the `SYSTEM` tab and make sure this dropdown or whatever:

<img width="787" alt="(this image here my friend)" src="https://user-images.githubusercontent.com/69256931/112609218-32a30780-8e13-11eb-9fe9-bef7184dbdb3.png">

looks like this:

<img width="784" alt="(dude get better wifi, or stop using the inspector lol)" src="https://user-images.githubusercontent.com/69256931/112609313-4fd7d600-8e13-11eb-897d-9d5931b886a9.png">

and then hit <img height="10" width="40" alt="(save)" src="https://user-images.githubusercontent.com/69256931/112609399-65e59680-8e13-11eb-8ba8-8f391ddf3aa6.png">

Hopefully it will work but I am on macOS so it'll be different for you.

**Comment by qdljerry** [comment]
Try to use the cdrom as removable device

**Comment by Rafael124615** [comment]
Same Here It Keeps Booting To SeaBIOS And l Keep Getting “No bootable device.” When Trying To Install MS-DOS 2.0
![IMG_0685](https://github.com/user-attachments/assets/27368e8e-535c-4a60-ac25-7980ff0a89d2)
